database/postgres/postgres.go

```diff
@@ -41,17 +41,40 @@ type Config struct {
 	StatementTimeout time.Duration
 }
 
+type SQLWrapper struct {
```

Comment:
Don't think we need this. e.g. `*sql.DB` should work
It's probably worth documenting that `*sql.DB` satisfies this interface the interface should be moved up a package to `migrate/database`